src/libs/Navigation/linkingConfig.js

```diff
@@ -8,6 +8,7 @@ export default {
         'https://www.expensify.cash',
         'https://staging.expensify.cash',
         'http://localhost',
+        'https://new.expensify.com',
```

Comment:
Same question here, do we also need `https://www.new.expensify.com`?

Reply:
I wasn't planning on adding it, since I think it's less likely somebody would type in `www.new`, but that does remind me that I need to add `staging.new`